### PR TITLE
docs: Adding reverse proxy and Let's Encrypt example

### PR DESCRIPTION
**Pull request by miohtama**
You can do both reverse proxy and getting a Let's Encrypt certificate in a single command.

### CODE REVIEW

File: src/docs/markdown/quick-starts/reverse-proxy.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ If you don't have permission to bind to low ports, you can proxy from a higher p
 
 Then make a request to [localhost](https://localhost) (or whatever address you specified in `--from`) to see it working!
 
+You can also automatically bind `caddy` reverse proxy to a public IP address and have a Let's Encrypt certificate issued for.
+This command will automatically issue a TLS certificate for `mywebsite.example.com` and start reverse proxy.
+
+<pre><code class="cmd bash">caddy reverse-proxy --from https://mywebsite.example.com --to 127.0.0.1:9000</code></pre>
```
Comment (mholt) [inline src/docs/markdown/quick-starts/reverse-proxy.md:36]:
```suggestion
If you have a public domain name to use, Caddy can serve your proxy over HTTPS using a trusted certificate automatically:

<pre><code class="cmd bash">caddy reverse-proxy --from example.com --to 127.0.0.1:9000</code></pre>
```

**Review by mholt — CHANGES_REQUESTED** [review]
Thanks; I think we can make it a little simpler though.

(And what's up with the change at the end of the file?)

File: src/docs/markdown/quick-starts/reverse-proxy.md (hunk 1 above)
Comment (francislavoie) [reply src/docs/markdown/quick-starts/reverse-proxy.md:36]: > (And what's up with the change at the end of the file?)

Some IDEs add or remove the extra newline at the end depending on settings, and it ends up rendering like that in github.

**Comment by mholt** [comment]
Closing due to inactivity

**Pull request closed without merging**